README.md

```diff
@@ -30,18 +30,18 @@ To activate `RSpec` integration, add this to the `spec_helper.rb` file:
 require 'rspec'
 require 'datadog/ci'
 
-Datadog.configure do |c|
-  # Only activates test instrumentation on CI
-  c.tracing.enabled = (ENV["DD_ENV"] == "ci")
-
-  # Configures the tracer to ensure results delivery
-  c.ci.enabled = true
-
-  # The name of the service or library under test
-  c.service = 'my-ruby-app'
-
-  # Enables the RSpec instrumentation
-  c.ci.instrument :rspec, **options
+# Only activates test instrumentation on CI
+if ENV["DD_ENV"] == "ci"
```

Comment:
Offtopic: what do you think of making this check `ENV["DD_ENV"] == "ci"` internally, and assume most clients want this? And then provide an option to override it, for the exceptional clients that don't want this.

Reply:
I don't think making internal assumptions on when to turn things on or off based on `DD_ENV` would make sense to me as a user. It's much more likely I'd end up wondering why things aren't working because I missed the part in the docs that says `DD_ENV` has to be set and equal to `ci` for things to work.

Reply:
agree with @romainkomorndatadog, there is a very fine line between doing "magic" and surprising customers with unexpected default behaviour. Many clients use "test" as a DD_ENV value for CI for example. Here in the docs we just want to point out that they should enable CI visibility mode only when running on CI to avoid local test runs to be reported.